components/combobox/index.css

```diff
@@ -0,0 +1,451 @@
+}
+
+@media (forced-colors: active) {
+  .spectrum-Combobox {
+    --highcontrast-combobox-focus-indicator-color: Highlight;
+    --highcontrast-combobox-border-color: ButtonText;
+    --highcontrast-combobox-font-color: CanvasText;
+    --highcontrast-combobox-font-color-disabled: GrayText;
+    --highcontrast-combobox-icon-color: ButtonText;
+    --highcontrast-combobox-background-color: Background;
+    --highcontrast-combobox-border-color-disabled: GrayText;
+  }
+
+  .spectrum-Inputgroup-textfield {
+    &:hover {
+      .spectrum-Textfield-input {
+        border-color: var(--highcontrast-combobox-border-color);
+
+        &::placeholder {
+          color: var(--highcontrast-combobox-border-color);
+        }
+      }
+    }
+  }
+}
+
+.spectrum-Combobox {
+  position: relative;
+
+  display: inline-flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+
+  min-inline-size: var(--mod-combobox-min-inline-size, var(--spectrum-combobox-min-inline-size));
+  block-size: var(--mod-combobox-block-size, var(--spectrum-combobox-block-size));
+
+  margin-block-start: var(--mod-combobox-spacing-label-to-combobox, var(--spectrum-combobox-spacing-label-to-combobox));
+
+  border-radius: var(--mod-combobox-border-radius, var(--spectrum-combobox-border-radius));
+  border-width: var(--mod-combobox-border-width, var(--spectrum-combobox-border-width));
+
+  .spectrum-Combobox-textfield {
+    .spectrum-Textfield-validationIcon {
+      position: absolute;
+      inset-block-start: var(--mod-combobox-block-spacing-edge-to-alert, var(--spectrum-combobox-block-spacing-edge-to-alert));
+      inset-block-end: var(--mod-combobox-block-spacing-edge-to-alert, var(--spectrum-combobox-block-spacing-edge-to-alert));
+
+      inline-size: var(--mod-combobox-icon-size, var(--spectrum-combobox-icon-size));
+      block-size: var(--mod-combobox-icon-size, var(--spectrum-combobox-icon-size));
+    }
+ 
+    &:focus, &:focus-visible, &.is-focused, &.is-keyboardFocused {
+      border: none;
+
+      &::after {
+        box-shadow: none;
+      }
+    }
+  }
+
+  .spectrum-Combobox-button:not(:disabled):not(.is-invalid) {
+    .spectrum-PickerButton-fill {
+      /* Subcomponent token override */
+      --spectrum-alias-infieldbutton-fill-border-color-default: var(--spectrum-combobox-border-color-default);
+      border-color: var(--highcontrast-combobox-border-color, var(--mod-combobox-border-color-default, var(--spectrum-alias-infieldbutton-fill-border-color-default)));
+    }
+
+    &:focus, &.is-focused, &.is-keyboardFocused {
+      .spectrum-PickerButton-fill:not(.is-invalid) {
+        border-color: var(--highcontrast-combobox-bordr-color, var(--mod-combobox--border-color-key-focus, var(--spectrum-alias-infieldbutton-fill-border-color-key-focus)));
+      }
+    }
+  }
+
+  & .is-open.spectrum-Popover {
+    transform: translateY(var(--mod-combobox-spacing-edge-to-menu, var(--spectrum-combobox-spacing-edge-to-menu)));
+  }
+}
+
+.spectrum-Combobox.is-invalid {
```

Comment:
these are all ok, but maybe we should nest those like
```SCSS
.spectrum-Combobox {
  .is-invalid {
     ...
  .is-disabled {

etc

Reply:
Updated in [0a65bda](https://github.com/adobe/spectrum-css/pull/1556/commits/0a65bdaef0ac0f34a4cad138f63387069bb1772a)